Java/cp/Stack_using_Queue.java

```diff
@@ -0,0 +1,94 @@
+
+/* Java Program to implement a stack using two queue */
```

Comment:
Add a description about the Code/Algorithm ?

Reply:
I have given the description. 
Java Program to implement a stack using two queue */